reports/proofofplay-report-form.twig

```diff
@@ -218,7 +218,7 @@
                                         <th>{% trans "Type" %}</th>
                                         <th>{% trans "Display ID" %}</th>
                                         <th>{% trans "Display" %}</th>
-                                        <th>{% trans "Campaign" %}</th>
+                                        <th>{% trans "Ad Campaign" %}</th>
```

Comment:
This is not necessarily the ad campaign - it could also be a campaign of type list.